src/khoj/routers/api_chat.py

```diff
@@ -1081,6 +1127,8 @@ def collect_telemetry():
                 query_images=uploaded_images,
                 tracer=tracer,
                 train_of_thought=train_of_thought,
+                attached_file_context=attached_file_context,
+                raw_attached_files=raw_attached_files,
```

Comment:
Why are both attached_file_context and raw_attached_files required? And if they are required, can they grouped together into single object (e.g a `query_files` TypedDict, Class) instead?

Reply:
`attached_file_context` is packaged neatly for consumption by the LLM, uniformly sent down the evaluation tree. `raw_attached_files` are used for storing the raw data in the conversation history.